src/app/backendError.ts

```diff
@@ -35,7 +37,13 @@ export type InsufficientMatchFundsError = BackendError &
   };
 
 export function errorDescription(error: BackendError): string {
-  return error.error.error.description || error.message || 'Sorry, something went wrong'
+  const errorDetail = error.error.error;
+
+  return errorDetail.description
+    || errorDetail.publicMessage
+    || errorDetail.message
+    || error.message
+    || 'Sorry, something went wrong'
```

Comment:
message now appears like so:

![image](https://github.com/user-attachments/assets/d15b52f2-4476-451c-a069-7aee0d96b1b8)

Without this change instead of saying "your card was decline a more technical message about an http response code.